Tests/AK/TestSIMD.cpp

```diff
@@ -31,3 +35,5 @@ TEST_CASE(exp_approximate)
     EXPECT(fabsf(result[2] - 1.82211880f) <= accuracy);
     EXPECT(fabsf(result[3] - 2.22554093f) <= accuracy);
 }
+
+#pragma GCC diagnostic pop
```

Comment:
Hm, this doesn't actually work, it just gives an error at `pop` time now:

```
/home/sergey/serenity/Tests/AK/TestSIMD.cpp: In function ‘AK::SIMD::f32x4 AK::SIMD::exp(f32x4)’:
/home/sergey/serenity/Tests/AK/TestSIMD.cpp:39:27: error: SSE vector return without SSE enabled changes the ABI [-Werror=psabi]
   39 | #pragma GCC diagnostic pop
      |                           ^
```

I guess perhaps we could skip the `push`/`pop` game altogether and just `ignore` it?